source/features/load-hidden-comments.tsx

```diff
@@ -0,0 +1,50 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	// Retrieve the form associated with the "X Hidden Items\nLoad More" button
+	const form = select('form.ajax-pagination-form');
+	if (form) {
+		// The first button, which has the text "X hidden items".
+		const button = select('button', form);
+		if (button) {
+			// Extract the number of hidden items from the button text
+			const numberString = button.textContent?.trim()?.split(' ')[0];
```

Comment:
We prefer code to throw instead of silently fail when our expectations aren't met. We _expect_ the count to be there. If it isn't, the destructuring will throw.

```suggestion
			const [itemCount] = button.textContent!.match(/\d+/)!;
```